action.yml

```diff
@@ -53,7 +53,11 @@ inputs:
     description: 'Expected SHA256 checksum of the Tailscale package. If not provided, it will be fetched automatically.'
     required: false
     default: ''
-
+  ping:
```

Comment:
@mcoulombe I used the name `ping` instead of `targets` as `target` can have various meanings and `ping` seems clear about what it's doing. If you feel strongly that we should preserve the name, I can change it.